apps/users/tests/test_views.py

```diff
@@ -741,6 +733,18 @@ def test_can_customize_account(self):
         self.assertEqual(self.customize_account_request["last_name"], user.last_name)
         self.assertEqual(self.customize_account_request["job_title"], user.job_title)
 
+    def test_can_customize_account_without_job_title(self):
+        user = self._create_authenticated_user()
+        no_job_title = {**self.customize_account_request}
+        del no_job_title["job_title"]
+        response = self.client.patch(
+            reverse("user-customize-account", args=(user.pk,)),
+            data={**no_job_title},
+        )
+        self.assertEqual(response.status_code, status.HTTP_204_NO_CONTENT)
+        self.assertEqual(self.customize_account_request["first_name"], user.first_name)
+        self.assertEqual(self.customize_account_request["last_name"], user.last_name)
```

Comment:
No reason you should know, but we should be using pytest-style asserts for any new/modified tests. E.g.:

```suggestion
        assert self.customize_account_request["last_name"] == user.last_name
```

Reply:
👍 Updated in 89350c52660c790fd8314009183b486e4bd65990